homework/week-0/pooja/src/nyc/c4q/poojawins/MainTextAdventure.java

```diff
@@ -0,0 +1,11 @@
+package nyc.c4q.poojawins;
+
+/**
+ * Created by pooja on 5/3/15.
+ */
+public class MainTextAdventure {
+
+    public static void main(String[] args) {
+
```

Comment:
Build me out a little! Some simple scenarios to test the classes you've made (I coded a very basic fight between two characters in a room).